lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -66,3 +65,25 @@
   padding: $nx-spacing-xl $nx-spacing-l $nx-spacing-md $nx-spacing-l;
   white-space: normal;
 }
+
+.nx-body--page-scrolling {
+  overflow: auto;
+
+  .nx-page {
+    display: block;
+  }
+
+  .nx-page-header {
+    position: fixed;
+    top: 0;
+    z-index: 1;
+  }
+
+  .nx-page-content {
+    margin: $main-header-height auto 0 auto;
```

Comment:
> This variable should be moved to `_nx-variables.scss` since it is used in multiple files

It is defined in `_variables.scss`

Reply:
> > This variable should be moved to `_nx-variables.scss` since it is used in multiple files
> 
> It is defined in `_variables.scss`

Ahhh, but there are two almost identical variables.

Reply:
> > > This variable should be moved to `_nx-variables.scss` since it is used in multiple files
> > 
> > 
> > It is defined in `_variables.scss`
> 
> Ahhh, but there are two almost identical variables.

https://github.com/sonatype/sonatype-react-shared-components/pull/38/commits/51aa7116370022ffe92073bff10d4af399db0599